src/aspire/utils/cell.py

```diff
@@ -1,6 +1,9 @@
 import numpy as np
 import logging
 
+# ugh circular depends
```

Comment:
noting that I need to think about how I want to deal with this.

Reply:
made cell.py the leaf by moving code relating to BlkDiagMatrix into blk_diag_matrix.py.